File: In_Javascript/ex1.js
// Logika bilan yechganda
function katta_s(x, y, z) {
    if (x > y && x > z) {
        return x;
    } else if (y > x && y > z) {
        return y;
    } else {
        return z;
    }
}

// funksiya bilan yechganda
function katta_s_2(x, y, z) {
   return Math.max(x, y, z);
}

// Oson yo'l bilan yechganda
function katta_s_3(x, y, z) {
    return (x > y && x > z) ? x : (y > x && y > z) ? y : z;
}

console.log(katta_s(10, 20, 30));
console.log(katta_s_2(10, 20, 30));
console.log(katta_s_3(10, 20, 30));

File: In_Ruby/ex1.rb
# Logika orqaliy yechganda.
def big_number(x, y, z)
  if x > y && x > z
    x
  else
    if y > z
      y
    else
      z
    end
  end
end

# Oson yo'l bilan yechgan.
def big_number_2(x, y, z)
  [x, y, z].max
end

puts big_number(1, 2, 3)

puts big_number_2(1, 2, 3)